Assignment/Program.cs

```diff
@@ -1,7 +1,17 @@
-﻿namespace Assignment {
-   internal class Program {
-      static void Main (string[] args) {
-         Console.WriteLine ("Hello world!");
+﻿for (; ; ) {
+   Console.Write ("Enter number to be checked :");
+   if (int.TryParse (Console.ReadLine (), out int num)) {
+      int count = 0;

```

Comment:
Why do you need this count?

Reply:
Printing the output based on the count
Example: Given number is 11
Here i'm checking from 2 to 10 and taking the count
Then comparing the stored count with the expected count